app/views/components/widget/_widget.css.erb

```diff
@@ -24,7 +24,7 @@
     padding-right: 20px;
 }
 
-#fba-modal-title {
+.fba-modal-title {
```

Comment:
I don't think you want this change. It makes the title look crammed up against the top on the legacy form:

<img width="756" alt="Screenshot 2024-08-20 at 5 23 21 PM" src="https://github.com/user-attachments/assets/b2b9530b-8ced-429b-8ec5-0c25c1fd56e7">

Reply:
looks like the h1 has less margin/padding than the h2. 
adjusted